polaris.shopify.com/playroom.config.js

```diff
@@ -33,12 +33,29 @@ module.exports = {
   port: 9000,
   openBrowser: false,
   paramType: 'search', // default is 'hash'
-  webpackConfig: () => ({
+  webpackConfig: ({production}) => ({
+    ...(!production
```

Comment:
Opting to do this, because ideally in prod we absolutely do not want people to access components or component changes that are yet unpublished. This aliasing would allow this to happen, as the environment would always be referencing the local directory. 

This allows us to preserve the desired behaviour: 
* Dev mode, test the sandbox with the latest code in the local repo 
* In prod, we should always only have access to the latest published package in our sandbox environment. 

cc @jesstelford  @qt314